amplify/backend/ts-code/src/api/GetItem.ts

```diff
@@ -67,25 +72,27 @@ export class GetItem {
             GetItem.NAME, this.metrics
         )
     }
-
-    private header(entry: MainSchema): string {
-        var tags: string[] = entry.tags ? entry.tags.values : []
-        return `name: ${entry.name}`
-            + `\ndescription: ${entry.description}`
-            + `\ntags: ${tags.toString()}`
-            + `\nitems:`
-    }
-
-    private item(entry: MainSchema, id: string) {
-        var batch: string[] = entry.items[id].batch ? entry.items[id].batch.values : []
-        return `\n  id: ${id}`
-            + `\n    owner: ${entry.items[id].owner}`
-            + `\n    borrower: ${entry.items[id].borrower}`
-            + `\n    batch: ${batch}`
-            + `\n    notes: ${entry.items[id].notes}`
-    }
 }
 
 interface ScratchInterface {
     key?: string
+}
+
+// Output Formatting Functions. Exporting them to be used for testing.
+
+export function header(entry: MainSchema): string {
+    var tags: string[] = entry.tags ? entry.tags.values : []
```

Comment:
Is there a reason why we're using var instead of let or const here?

Reply:
Because I wrote this when I had about 1 day of javascript experience.